spec/diagrams/buffer-map-unmap.mmd

```diff
@@ -0,0 +1,12 @@
+sequenceDiagram
+    Note over Content timeline: mapState: "unmapped"
+    Note over Device timeline: [[state]]: "unmapped"
+    Content timeline ->> Device timeline: mapAsync()
+    Note over Content timeline: mapState: "pending"
+    Note over Device timeline: [[state]]: "mapping pending"
+    Device timeline ->> Content timeline: mapAsync() response
+    Note over Device timeline: [[state]]: "mapped"
+    Note over Content timeline: mapState: "mapped"
+    Content timeline ->> Device timeline: unmap()
+    Note over Content timeline: mapState: "unmapped"
+    Note over Device timeline: [[state]]: "unmapped"
```

Comment:
Can't comment on the images, but you might want to check the colours with an accessibility contrast checker. The `mapAsync` to me looks ... fuzzy?

Reply:
It's fuzzy for me too but I don't know what to do about it :( Maybe use a light theme and white background?

Reply:
Yea, maybe try the light version and see if it's better?

Reply:
Turns out the theme is setting both the fill color and the stroke color of the text, which makes it look bad. It would be one thing if they were both different colors - maybe it would make it work on a transparent background - but it's setting them to the same color. Which for whatever reason makes it look blurry.